Example/GiniVision/DocumentService.swift

```diff
@@ -82,14 +93,6 @@ final class DocumentService {
         print("Gini Vision Library for iOS (\(GiniVision.versionString)) / Client id: \(clientId)")
     }
     
-    func populateSettingsPage() {
```

Comment:
No settings page anymore?

Reply:
No one was using it, it wasn't even documented. And adding a password in the settings is not the best way I think.